### PR TITLE
Play game command

### LINKED ISSUE #260: Can play game
The user might mistype a command, and then the program closes with an error. Maybe we should let this happen.
**SOLUTION**: Make a new command in InputOutputController for the user called `play` in that case the InputOutputController won't close the program in case of an invalid command (this should also switch off the Logger, like `echo_off`).

### PR DESCRIPTION
**Pull request by poiuztrewq1**
- Done the play game command. If the user is in play mode and the command is not recognized, it won't exit. (first commit)
- I thought we could do something really fancy and searched for an algorithm to get the closest matching command from a list. So when somebody types `caus_sunstrom` the program will ask `Did you mean cause_sunstorm?` 

Extra: Apart from `play` I added a `generate` command, so users can go into 'assisted' mode with their own created map.

closes #260 #280

### CODE REVIEW

**Review by triszt4n — CHANGES_REQUESTED** [review]
Love it, Levenstein's is a nice feature. 🚀 

Answers to questions:
1. Feature can stay.
2. Right now the `Handle` method returns if interpreting the command was successful, and in every case we override `isRunning` with the return value. You can introduce a new variable `isInterpreted`, and use it to store the return value in that. One more thing: the `while` cycle should consider this variable when in play mode. This way the code below can stay as is, and we need a new conditional outside the `default` case and run the conditional everytime a line is read and `isPlayMode && !isInterpreted`.

File: java/src/hu/holyoil/controller/InputOutputController.java [hunk 1]
```diff
@@ -96,9 +116,21 @@ public void ParseCommand(InputStream inputStream) {
...
-                    isRunning = false;
+                    if (isPlayMode) {
+                        System.out.println("Command not recognised.");
+                        System.out.println("For all available commands refer to the documentation");
+                        String closest = Collections.min(commands, Comparator.comparingInt(s -> Distance(s, command[0])));
+                        System.out.println("\t Did you mean: " + closest + " ?");
+                    } else {
+                        System.out.println("Command not recognized: " + line.split(" ")[0]);
+                        isRunning = false;
+                    }
```
Comment (triszt4n) [inline java/src/hu/holyoil/controller/InputOutputController.java:133]: This

**Comment by vercibar** [comment]
Shouldn't this command also generate the world using GameController.StartGame()?

**Comment by poiuztrewq1** [comment]
> 
> 
> Shouldn't this command also generate the world using GameController.StartGame()?

Well, can do, but please refer to the issue of the PR:
```
The user might mistype a command, and then the program closes with an error. Maybe we should let this happen.
SOLUTION: Make a new command in InputOutputController for the user called play in that case the InputOutputController won't close the program in case of an invalid command (this should also switch off the Logger, like echo_off).
```

**Review by triszt4n — APPROVED** [review]
# Nice 🥇

**Pull request merged**